core/src/main/scala/higherkindness/mu/rpc/srcgen/proto/gen.scala

```diff
@@ -0,0 +1,33 @@
+package higherkindness.mu.rpc.srcgen.proto
+
+import protocbridge.Artifact
+import protocbridge.SandboxedJvmGenerator
+import higherkindness.mu.rpc.srcgen.Model
+
+object gen {
+
+  /**
+   * The entrypoint to our protoc plugin. This is what is called from the sbt plugin.
+   */
+  def apply(
+      idiomaticEndpoints: Boolean,
+      compressionType: Model.CompressionTypeGen
+  ): (SandboxedJvmGenerator, Seq[String]) =
+    (
+      SandboxedJvmGenerator.forModule(
+        "scala",
+        Artifact(
+          higherkindness.mu.rpc.srcgen.BuildInfo.organization,
+          "mu-srcgen-core_2.12",
```

Comment:
I thought so too, but actually we want to hardcode this to _2.12 because this code is called from an sbt plugin. The `sbt-mu-srcgen` plugin and the `mu-srcgen-core` library are only published for 2.12 because that's the Scala version that sbt uses.

Reply:
Yes, my idea was to make the transition to a new Scala version in SBT easier. It's not changed frequently, but I imagine this code can be forgotten once the change is needed.

Reply:
Updated so we now use sbt-buildinfo to generate all the information we need.